selene_sdk/predict/model_predict.py

```diff
@@ -531,19 +523,21 @@ def get_predictions_for_fasta_file(self,
 
 
     def get_predictions(self,
-                        input_path,
-                        output_dir,
+                        input,
+                        output_dir=None,
                         output_format="tsv",
                         strand_index=None):
         """
-        Get model predictions for sequences specified in a FASTA or BED file.
+        Get model predictions for sequences specified as a raw sequence,
+        FASTA, or BED file.
 
         Parameters
         ----------
-        input_path : str
-            Input path to the FASTA or BED file.
-        output_dir : str
-            Output directory to write the model predictions.
+        input : str
+            A single sequence, or a path to the FASTA or BED file input.
+        output_dir : str, optional
+            Output directory to write the model predictions. If this is left
```

Comment:
Add default to the description. (`Default is None. Output directory to...`) Maybe also clarify that FASTA and BED inputs **require** an output directory to be specified?